crypto/bip32.c

```diff
@@ -294,8 +294,12 @@ int hdnode_public_ckd_cp(const ecdsa_curve *curve, const curve_point *parent,
     hmac_sha512(parent_chain_code, 32, data, sizeof(data), I);
     bn_read_be(I, &c);
     if (bn_is_less(&c, &curve->order)) {  // < order
-      scalar_multiply(curve, &c, child);  // b = c * G
-      point_add(curve, parent, child);    // b = a + b
+      // b = c * G
+      uint8_t child_pubkey[65] = {};
```

Comment:
nit: empty initializer list is not valid until C23, while our build scripts claim to be using gnu11 and gnu99, although gcc seems to accept it without complaints